rcgen/src/key_pair.rs

```diff
@@ -165,13 +167,15 @@ impl KeyPair {
 
 	/// Parses the key pair from the ASCII PEM format
 	///
-	/// The key must be a DER-encoded plaintext private key; as specified in PKCS #8/RFC 5958;
+	/// If `aws_lc_rs` feature is used, then the key must be a DER-encoded plaintext private key; as specified in PKCS#1/RFC 3447, RFC 5915, or PKCS #8/RFC 5958;
```

Comment:
two small suggestions:

1. Let's order this PKCS8, SEC1, PKCS1 - I think that's ordering it most common to least common. (Especially since only PKCS8 was supported before).
2. Let's mention "SEC1" alongside RFC 5915 since I think people will be pattern matching on the first more than reading the linked RFC.

```suggestion
	/// If `aws_lc_rs` feature is used, then the key must be a DER-encoded plaintext private key; as specified in PKCS #8/RFC 5958, SEC1/RFC 5915, or PKCS#1/RFC 3447;
```